test/e2e/suite/common/common.go

```diff
@@ -0,0 +1,36 @@
+package common
+
+const (
+	SignSuccessfully   = "Successfully signed"
+	VerifySuccessfully = "Successfully verified"
+	VerifyFailed       = "signature verification failed"
+)
+
+const (
+	// HeaderVerificationPlugin specifies the name of the verification plugin that should be used to verify the signature.
+	HeaderVerificationPlugin = "io.cncf.notary.verificationPlugin"
+
+	// HeaderVerificationPluginMinVersion specifies the minimum version of the verification plugin that should be used to verify the signature.
+	HeaderVerificationPluginMinVersion = "io.cncf.notary.verificationPluginMinVersion"
+)
+
+// Capability is a feature available in the plugin contract.
+type Capability string
+
+const (
+	// CapabilitySignatureGenerator is the name of the capability
+	// for a plugin to support generating raw signatures.
+	CapabilitySignatureGenerator Capability = "SIGNATURE_GENERATOR.RAW"
+
+	// CapabilityEnvelopeGenerator is the name of the capability
+	// for a plugin to support generating envelope signatures.
+	CapabilityEnvelopeGenerator Capability = "SIGNATURE_GENERATOR.ENVELOPE"
+
+	// CapabilityTrustedIdentityVerifier is the name of the
+	// capability for a plugin to support verifying trusted identities.
+	CapabilityTrustedIdentityVerifier Capability = "SIGNATURE_VERIFIER.TRUSTED_IDENTITY"
+
+	// CapabilityRevocationCheckVerifier is the name of the
+	// capability for a plugin to support verifying revocation checks.
+	CapabilityRevocationCheckVerifier Capability = "SIGNATURE_VERIFIER.REVOCATION_CHECK"
+)
```

Comment:
Yes, they are dumplicated code. However, as an integration test, I think any constant defined in the source code should be not used by testing code. Because the source code may be changed accidently, and the test may not detect the issue if we use the constant in the source code directly.